### PR TITLE
fix(links): Updates link colors and hover/active states

### PR DESCRIPTION
**Pull request by kennyw12**
Paired with Justin on this.
Removes link-lowlight-alt

[Finishes #100253032]

### CODE REVIEW

File: src/pivotal-ui/components/links/links.scss [hunk 1]
```diff
@@ -49,30 +48,32 @@ a{
...
     text-decoration: none;
+    color: lighten($blue-2, 5%);
+  }
+
+  &.active{
+    color: darken($blue-2, 5%);
   }
 
   &.link-text {
-    &:not(.link-lowlight):hover {
```
Comment (gpleiss) [inline src/pivotal-ui/components/links/links.scss]: @kennyw12 was this change here enough to fix hover styles for **all** links? I know that bootstrap by default has an underline on links. Did you inspect all the links to ensure the hover style was fixed for everything?

Comment (gpleiss) [inline src/pivotal-ui/components/links/links.scss]: Ah just noticed the `text-decoration: none` on L:50. Nevermind!

**Comment by gpleiss** [comment]
:+1: looks good to me. Ping @stubbornella

**Comment by stubbornella** [comment]
Looking at it now...

File: src/pivotal-ui/components/links/links.scss [hunk 2]
```diff
@@ -49,30 +48,32 @@ a{
...
   &:hover {
-    color: $link-lowlight-hover-color;
+    color: lighten($link-lowlight-color, 5%);
   }
-}
-
-.link-lowlight-alt {
-  color: $link-lowlight-alt-color;
 
-  &:hover {
```
Comment (stubbornella) [inline src/pivotal-ui/components/links/links.scss]: Do we not want a hover color change for `link-lowlight-alt`?

**Comment by stubbornella** [comment]
Anywhere we've defined `:hover` we need to also define `:focus` and `:active` pseudo states for accessibility purposes. `:focus` is what the link looks like when you tab to it with keyboard navigation. `:active` is what it looks like when it has been activated, but the corresponding page hasn't loaded yet.

File: src/pivotal-ui/components/links/links.scss [hunk 3]
```diff
@@ -49,30 +48,32 @@ a{
...
+
 }
 
 
+
 .link-lowlight {
   font-weight: $link-lowlight-font-weight;
   color: $link-lowlight-color;
 
   &:hover {
```
Comment (stubbornella) [inline src/pivotal-ui/components/links/links.scss]: should probably be:

```
&:hover, &:focus {
```

File: src/pivotal-ui/components/links/links.scss [hunk 4]
```diff
@@ -80,7 +81,11 @@ a{
   color: $link-inverse-color;
 
   &:hover {
```
Comment (stubbornella) [inline src/pivotal-ui/components/links/links.scss]: Should probably be:

```
&:hover, &:focus {
```

File: src/pivotal-ui/components/links/links.scss (hunk 2 above)
Comment (stubbornella) [inline src/pivotal-ui/components/links/links.scss]: Maybe the default link hover color is what you also wanted for the lowlight link?

**Comment by atomanyih** [comment]
@kennyw12 Could you address the conflicts on this branch? :fallen_leaf:

**Comment by kennyw12** [comment]
Created a separate PR so no force pushing.
https://github.com/pivotal-cf/pivotal-ui/pull/236

**Pull request closed without merging**